python-checks/src/main/java/org/sonar/python/checks/regex/VerboseRegexCheck.java

```diff
@@ -41,15 +44,16 @@ public class VerboseRegexCheck extends AbstractRegexCheck {
   private static final String ISSUE_MESSAGE_PATTERN = ".+syntax '(.+)' instead of.+";
   private static final Pattern issueMessagePattern = Pattern.compile(ISSUE_MESSAGE_PATTERN);
   public static final String QUICK_FIX_FORMAT = "Replace with \"%s\"";
+  public static final String REDUNDANT_RANGE_MESSAGE = "Use simple character '%s' instead of '%s'.";
```

Comment:
I'd suggest `Replace with '%s'` for the issue message.

Reply:
discussed, not needed